File: Backend/Backend/Model/UserVehicle.cs
using System;
using System.Collections.Generic;

namespace Backend.Model;

public partial class UserVehicle
{
    public int VehicleId { get; set; }

    public int UserId { get; set; }

    public string? Name { get; set; }

    public string? Description { get; set; }

    public string? Rcnumber { get; set; }

    public string? Model { get; set; }

    public string? Video { get; set; }
}

File: Frontend/src/app/model/userVehicle.ts
export class UserVehicle{
    public vehicleId: number;
    public userId: number;
    public name: string;
    public description: string;
    public rcnumber: string;
    public model: string;
    public video: string;

    constructor(vehicleId: number, userId: number, name: string, description: string, rcnumber:string, model: string, video: string){
        this.vehicleId = vehicleId;
        this.userId = userId;
        this.name = name;
        this.description = description;
        this.rcnumber = rcnumber;
        this.model = model;
        this.video = video;
    }
}